File: 2_PhoneBook/Flutter_PhoneBook/lib/domain/item.dart
class Item {
  final int? id;
  final String? name;
  final String? tel;

  Item({this.id, this.name, this.tel}); // 선택적 매개변수 {} 안에 넣으면 받아도 되고 안받아도 된다.

  // 통신을 위해서 json 처럼 만들어주기
  Item.fromJson(Map<String, dynamic> json)
      : id = json["id"],
        name = json["name"],
        tel = json["tel"];
}

File: 2_PhoneBook/SpringServer/flutterPhoneBook/src/main/java/com/cos/flutterPhoneBook/domain/Item.java
package com.cos.flutterPhoneBook.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Builder
@Data
@Entity

public class Item {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String name;
	private String tel;
}
